src/metabase/api/email.clj

```diff
@@ -69,24 +68,18 @@
   [:as {settings :body}]
   {settings su/Map}
   (api/check-superuser)
-  (let [email-settings (select-keys settings (keys mb-to-smtp-settings))
-        smtp-settings  (-> (set/rename-keys email-settings mb-to-smtp-settings)
-                           (assoc :port (some-> (:email-smtp-port settings) Integer/parseInt)))
-        ;; TODO - this is :thumbs_down`, we should just use `with-redefs` for `email/test-smtp-connection` where
-        ;; needed in the tests. FIXME!
-        response       (if-not config/is-test?
-                         ;; in normal conditions, validate connection
-                         (email/test-smtp-connection smtp-settings)
-                         ;; for unit testing just respond with a success message
-                         {:error :SUCCESS})
```

Comment:
Tweaked to use `with-redefs` in the test itself. Not strictly needed, but makes this test pass from the REPL